tests/flow/tests.sh

```diff
@@ -0,0 +1,268 @@
+#!/bin/bash
+
+# [[ $VERBOSE == 1 ]] && set -x
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+ROOT=$(cd $HERE/../.. && pwd)
+READIES=$ROOT/deps/readies 
+. $READIES/shibumi/defs
+
+VALGRIND_REDIS_VER=6.2
```

Comment:
wouldn't this become a headache to keep track of ?

Reply:
Inject from the Makefile, same as used for docker builds?

Reply:
We don't have one global place in which we state the redis version we use. Here, as we're building a valgrind-comparible redis, we just to be good enough, so we take 6.2. We can also say just `6`, which will yield the same result.
I don't think it creates a maintenance problem (I will go with the `6` thing, though).